### PR TITLE
Implement Search & Filter #25

### PR DESCRIPTION
**Pull request by VivianObiako**
implemented React-Table library
integrated a search function by name.

### CODE REVIEW

**Comment by braginini** [comment]
Great work, @VivianObiako ! 
 I left a few suggestions. 
 
 Also, I think positioning elements the way shown below is better - search bar to the left and [All|Online] and [Add Peer] to the right
 
https://ibb.co/d7jBg9q

**Comment by VivianObiako** [comment]
@braginini , Thanks for your feedback.
Yeah, I understand. I thought there need only search function by name.
Regarding to the filtering,I will make it by name. right?

**Comment by braginini** [comment]
@VivianObiako I misinterpreted some things. 
Filters are fine, sorry!

What is missing is sorting. Basically, the functionality to choose to sort by filed (Name and Last Seen both ascending and descending). Selection of the soring can be done as a separate dropdown or right on the column.

**Review by braginini — CHANGES_REQUESTED** [review]

**Comment by VivianObiako** [comment]
Hello, @braginini 
I have implemented your feedback:
- the case insensitive search
- sort by name and last seen

**Review by braginini — APPROVED** [review]

File: src/views/Peers.js [hunk 1]
```diff
@@ -1,248 +1,511 @@
...
+                          className="rounded p-2 border border-gray-300 focus:border-gray-400 outline-none w-[200px]"
+                          onChange={(e) => sortTable(e.target.value)}
+                        >
+                          <option value={0}>Name: Ascending</option>
+                          <option value={1}>Name: Descending</option>
+                          <option value={2}>Last Seen: Ascending</option>
+                          <option value={3}>Last Seen: Descending</option>
+                        </select>
+                      </div>
+                    </div>
```
Comment (braginini) [inline src/views/Peers.js:278]:
```suggestion
    <div className="flex">
                      <input
                        className="text-sm rounded p-2 border border-gray-300 focus:border-gray-400 outline-none w-[300px]"
                        placeholder="Search..."
                        type="search"
                        onChange={(e) => handleSearch(e.target.value)}
                      />
                      <div className="flex items-center">
                        <p className="ml-6 text-sm text-gray-700">Sort by: &nbsp;</p>
                        <select
                          className="bg-gray-50 text-sm text-gray-500 rounded p-2 border border-gray-300 focus:border-gray-400 outline-none"
                          onChange={(e) => sortTable(e.target.value)}
                        >
                          <option className="text-sm text-gray-500" value={0}>Name: Asc</option>
                          <option className="text-sm text-gray-500" value={1}>Name: Desc</option>
                          <option className="text-sm text-gray-500" value={2}>Last Seen: Asc</option>
                          <option className="text-sm text-gray-500" value={3}>Last Seen: Desc</option>
                        </select>
                      </div>
                    </div>
```

Comment (braginini) [reply src/views/Peers.js:278]: Minor styling changes to apply

**Review by braginini — CHANGES_REQUESTED** [review]

**Comment by braginini** [comment]
Awesome!
Merging. Thank you @VivianObiako

**Review by braginini — APPROVED** [review]

**Comment by VivianObiako** [comment]
Great, it's my pleasure.

**Pull request merged**